src/components/Camera/style.scss

```diff
@@ -112,8 +112,33 @@
   }
 
   .docLiveCaptureFrame & {
-    height: auto;
-    margin: 0;
+    left: 0;
+
+    @media (orientation: portrait) {
+      height: 100%;
+      width: 100%;
+    }
+
+    @media (orientation: landscape) {
+      height: auto;
+      width: 100%;
+      top: 21.5%;
+    }
+
+    @media (--small-viewport) and (--below-high-pixel-density) {
+      height: auto;
+      width: 100%;
+      top: -34%;
+      margin: 50% auto;
+    }
+
+    @media (--high-pixel-density) and (orientation: portrait) {
+      top: -20%;
+    }
+
+    @media (--high-pixel-density) and (orientation: landscape) {
+      top: 0;
+    }
```

Comment:
Why do we need to do all this when for selfie we can just use `object-fit: cover`?

Reply:
Good question. The implementation used to be just using `object-fit: cover` similar to selfie but was changed earlier this year to fix an issue with zoomed in live document capture view on the higher spec Huawei phones.

Reply:
I wonder why those Huawei phones don't have the zooming issue on selfie. Maybe because of multiple environmental cameras?
Anyways, I can see that the live document capture is still broken on desktop and although the feature is not meant to be used on desktop, we can still fix it by using `object-fit: cover` for desktop only?

Reply:
Yeah, I think so too (re: Huawei phones). Since only thing I can think of is the selfie isn't using the same back camera. 

Will add the adjustment to use `object-fit: cover` for desktop only. 👍